Symfony/CS/Fixer/Contrib/PhpdocSingleLineVarSpacingFixer.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Fixer for part of rule defined in PSR5 ¶7.22.
+ *
+ * @author SpacePossum
+ */
+final class PhpdocSingleLineVarSpacingFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        /** @var Token $token */
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            $content = $token->getContent();
+            $content = preg_replace_callback(
+'               #^/\*\*[ \t]*@var[ \t]+(\S+)[ \t]+(\$*\S+)?[ \t]+([^\n]*)\*/$#',
```

Comment:
- no need for spaces before `#`
- missing utests without space before `@` (`/**@var Foo*/`)
- I think we don't want to fix invalid docs with `$$$$$$$$var_name` in second capturing group

(btw, please move to Symfony level)